CDCgov/NEDSS-Modernization
# Added Table Component consistency for Patient Morbidity Report

## Description

Table Component Consistency for Patient Morbidity Reports

## Tickets

https://cdc-nbs.atlassian.net/browse/CNFT1-1714

## Steps to verify
1. Log in
2. Navigate to advanced search
3. Search for any patient
4. Navigate to patient profile and navigate to Events tabs and observe the Morbidity Reports table functioning as expected.

## Checklist before requesting a review
